h/services/jwt.py

```diff
@@ -67,9 +80,7 @@ def __init__(self, jwt_signing_key):
         self.jwt_signing_key = jwt_signing_key
 
     @classmethod
-    def decode_oidc_idtoken(
-        cls, token: str, key_set_url: str, algorithms: list[str]
-    ) -> dict[str, Any]:
+    def decode_oidc_idtoken(cls, token: str, keyset_url: str) -> dict[str, Any]:
```

Comment:
I couldn't resist renaming `key_set_url` to `keyset_url` for consistency with `keyset_url` and `KEYSET_URL` names used elsewhere. Also `key_set_url` just hurts my eyes, this would _absolutely_ be `keyset_url` in Pythonic naming style.

`keyset_url` makes it clear that it's the URL for the key set. Whereas `key_set_url` is more ambiguous. Does that mean `keyset_url` or `key_seturl` or indeed `key_set_url`? Is the `set` a verb or a noun? Etc.

Some programmers like to religiously separate all words with underscores in variable names without thinking about it but this is not good, readable or idiomatic naming style in Python. `keyset_url` is much more readable than `key_set_url`.